### ISSUE TITLE
Synthesize fails on mono tracks.

### ISSUE THREAD

**Issue by tkell** [algorithmic-music-exploration/amen#32]
As per @bmcfee:

Ok, my original bad test case works now, but this one fails

```
→  python reverse.py  ~/data/CAL500/mp3/art_tatum-willow_weep_for_me.mp3 
Traceback (most recent call last):
  File "reverse.py", line 19, in <module>
    out = synthesize(beats)
  File "/home/bmcfee/git/amen/amen/synthesize.py", line 64, in synthesize
    resampled_audio, left_offsets, right_offsets = time_slice.get_samples()
  File "/home/bmcfee/git/amen/amen/time.py", line 32, in get_samples
    left_offsets, right_offsets = self._get_offsets(starting_sample, ending_sample)
  File "/home/bmcfee/git/amen/amen/time.py", line 46, in _get_offsets
    zero_crossings = librosa.zero_crossings(channel)
  File "/home/bmcfee/git/librosa/librosa/core/audio.py", line 526, in zero_crossings
    y[np.abs(y) <= threshold] = 0
TypeError: 'numpy.float32' object does not support item assignment
```

Can we expand the test fixtures here to have both stereo and mono examples?

(Yes, yes we can.)

**Comment by tkell** [comment]
I would like to take mono input and return stereo output.  Is that sane?

**Comment by tkell** [comment]
Started on this with #33.

**Comment by bmcfee** [comment]
> I would like to take mono input and return stereo output. Is that sane?

Yeah.  I remember noodling this out with @blacker, summarized [here](https://github.com/algorithmic-music-exploration/amen/issues/5#issuecomment-109674918).

**Comment by tkell** [comment]
Got it, thanks!

I am currently making `synthesize` turn everything into stereo, but I will add a `mono` flag to `Audio.to_wav` so people who really, really want mono output can get mono output.

**Comment by tkell** [comment]
I think I've got this with #33.  I am not going to do the `mono` flag in to_wav as part of this, but I'll get to it soon.  

Please to send comments & special magic!

**Comment by tkell** [comment]
Fixed with #33!
